content/en/cosign/keyless.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Keyless Signatures"
+category: "Cosign"
+position: 122
+---
+
+You can review the [full design document](https://docs.google.com/document/d/1461lQUoVqbhCve7PuKNf-2_NfpjrzBG5tFohuMVTuK4/edit#) for more details.
+
+This document explains how the keyless signatures work in Cosign. This signature mode relies on the Sig store Public Good Instance.
+
+The following examples use this image:
+
+```shell
+$ IMAGE=gcr.io/dlorenc-vmtest2/demo
+$ IMAGE_DIGEST=$IMAGE@sha256:97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36
+```
+
+## Usage
+
+Keyless signing:
+
+```shell
+$ COSIGN_EXPERIMENTAL=1 cosign sign $IMAGE_DIGEST
+Generating ephemeral keys...
+Retrieving signed certificate...
+Your browser will now be opened to:
+https://accounts.google.com/o/oauth2/v2/auth?access_type=offline&client_id=&redirect_uri=http%3A%2F%2F127.0.0.1%3A5556%2Fauth%2Fgoogle%2Fcallback&response_type=code
+Pushing signature to: gcr.io/dlorenc-vmtest2/demo:sha256-97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36.sig
+```
+
+Keyless verifying:
+
+```shell
+$ COSIGN_EXPERIMENTAL=1 cosign verify $IMAGE
+The following checks were performed on all of these signatures:
+  - The cosign claims were validated
+  - The claims were present in the transparency log
+  - The signatures were integrated into the transparency log when the certificate was valid
+  - Any certificates were verified against the Fulcio roots.
+Certificate subject:  dlorenc@google.com
+{"Critical":{"Identity":{"docker-reference":""},"Image":{"Docker-manifest-digest":"sha256:97fc222cee7991b5b061d4d4afdb5f3428fcb0c9054e1690313786befa1e4e36"},"Type":"cosign container image signature"},"Optional":null}
+```
+
+That's it! No keys!
+The rest of the flags (annotations, claims, tlog, etc.) should all work the same.
+
+## Overview
+
+This uses ephemeral keys and certificates, which are signed automatically by the `fulcio` root CA.
+Signatures are stored in the `rekor` transparency log, which automatically provides an attestation
+as to when the signature was created.
+
+Information on the `fulcio` root CA can be found in the [fulcio repository](https://github.com/sigstore/fulcio).
+
+### Keys
+
+The root CA keys are hard-coded in Cosign today.
```

Comment:
This section is out of date, delete. A TODO could be adding more information about TUF.